phase1/gce/configure-vm-kubeadm.sh

```diff
@@ -51,15 +55,19 @@ fi
 
 case "${ROLE}" in
   "master")
-    OPTS=''
+    ADVERTISE_ADDRESS=$(get_metadata "k8s-advertise-addresses")
+    PARAMS="--token ${TOKEN} --apiserver-bind-port 443 --apiserver-advertise-address ${ADVERTISE_ADDRESS}"
```

Comment:
nit: shouldn't have both PARAMS and OPTS, choose one

Reply:
OPTS are the options that do not get saved and passed to kubadm init later. PARAMS are saved and passed on. I do need to separate them. Is there perhaps a better name choice or something that would make it more clear.